server/tserver/src/main/java/org/apache/accumulo/tserver/scan/LookupTask.java

```diff
@@ -87,23 +85,23 @@ public void run() {
       Key partNextKey = null;
       boolean partNextKeyInclusive = false;
 
-      Iterator<Entry<KeyExtent,List<Range>>> iter = session.queries.entrySet().iterator();
+      // Iterator<Entry<KeyExtent,List<Range>>> iter = session.queries.entrySet().iterator();
+      log.error("number of extents: " + session.queries.size() + " -- " + session.queries.keySet());
```

Comment:
Is this an error? If so, could the message also include what the error / issue is?

Reply:
I decided that adding Pair<KeyExtent, List<Range>> of an unfinished Range back to the front of the LinkedList ensures that it gets handled next.  That means that if the LookupTask ends due to time or result size then a record of the last partial scan will get returned.  If the unfinished ranges get fully scanned, then the KeyExtent will be recorded as a full scan and the record of a partial scan will be removed.  This seems to make the most sense.